# Gene Expression in Lung Squamous Cell Carcinoma (LUSC)

### Author: Logan Correa

## Abstract
Lung squamous cell carcinoma (LUSC) remains a significant health concern globally, with high mortality rates and heterogeneous genetic profiles. This study investigates the predictive potential of specific gene mutations associated with LUSC development. Utilizing a cohort of 551 patients sourced from the Genomic Commons Data Portal TCGA-LUSC project, comprising 56,907 gene expression transcripts, we applied comprehensive analyses integrating clinical, genomic, and environmental factors. Data preprocessing involved logarithmic transformation, scaling, and filtering based on frequently mutated genes associated with lung cancer. A volcano plot and heatmap visualization highlighted significant gene expression changes and distinct separation between normal and tumor patients. Principal Component Analysis (PCA) facilitated dimensionality reduction, revealing significant variance captured by key genes such as BUB1B, HMGA1, KNSTRN, EZH2, and FEN1. Logistic regression models utilizing PCA components indicated that PC1 alone provided sufficient information for predictive modeling, demonstrating a statistically significant association with LUSC diagnosis. Higher values of PC1 correlated with increased odds of LUSC diagnosis, suggesting its potential as a predictive biomarker. These findings underscore the importance of integrating genomic data for enhanced understanding and prediction of LUSC progression and treatment outcomes, but further research is needed to explore the functional implications of the identified genes and their potential as therapeutic targets in LUSC.

## Introduction
According to the National Cancer Institute, approximately 40.5% of men and women will be diagnosed with cancer at some point during their life with 12% of these cases consisting of lung and bronchus cancers [1]. Lung squamous cell carcinoma (LUSC), a subtype of non-small cell lung cancer, compromises nearly 30% of all lung cancer cases and is a significant global health concern due to its high mortality rates [2]. 

Prior research has emphasized the heterogeneous nature of LUSC, underscoring its diverse genetic and molecular profiles [3]. While previous findings have identified key genes such as  FLRT3, PPP2R2C, and MMP3; challenges persist in predicting patient responses and understanding the factors contributing to disease aggressiveness [4].

Recent literature also underscores the importance of comprehensive analyses that integrate clinical, genomic, and environmental factors to enhance our understanding of LUSCs progression and treatment outcomes [5]. This study aims to address these challenges by investigating how mutations in specific genes associated with LUSCs can serve as predictors of cancer development. We hypothesize that there will be a clear distinction between normal and tumor gene expression and that mutations in specific genes linked to LUSCs can function as predictive indicators of cancer development.

## Methods
The clinical dataset for LUSCs was sourced from the Genomic Commons Data Portal TCGA-LUSC project [6], encompassing a cohort of 551 patients. Each patient's profile includes 56,907 distinct gene expression transcripts, normalized by TPM (transcripts per kilobase million). Notably, the dataset exhibits an imbalance, with 49 patients diagnosed as healthy (normal), while the remaining 502 patients are diagnosed with cancer (tumor).

The initial phase of this study involved conducting data transformation and scaling on the gene expression data. Logarithmic transformation was applied to normalize skewed distributions, equalize variances across genes, and improve interpretability by making fold changes more intuitive. These preprocessing steps assist in meeting statistical assumptions such as normality and equal variances, thereby enhancing the effectiveness of subsequent analyses and machine learning algorithms. Scaling the data further diminishes the influence of outliers, ensuring that individual genes do not unduly dominate the analysis solely based on their expression levels. Data was scaled using the R ‘scale()’ function to standardize numeric data. This standardization involves transforming the data so that it has a mean of zero and a standard deviation of one. 

The dataset underwent filtering using a curated list of frequently mutated genes associated with lung cancer sourced from the GCD Portal [7]. This filtering process reduced the number of genes of interest from over 56,000 to approximately 700. Additionally, patients displaying gene expression identified as outliers (below Q1 - 4 x IQR and above Q3 + 4 x IQR) were excluded from the analysis.

A volcano plot was generated to depict gene expression log2 folds against log10 t-test p-values, illustrating significant downregulation, upregulation, and nonsignificant changes in gene expression. Genes lacking significant regulation (α = 7.1e-5, Bonferroni-corrected) were excluded from downstream analysis. In addition to this, a heatmap was created using a random sample of patients (25 normal, 25 tumor) to visualize gene expression regulation for the top 20 most statistically significant genes identified in the volcano plot.

Principal Component Analysis (PCA) was conducted on the dataset to diminish the dimensionality of the independent variables, which represent genes. Multiple Logistic regression models were developed using the PCA components to analyze the association between the reduced set of gene expression variables and the diagnosis of normal lung tissue versus lung tumor tissue in cases of lung cancer. This approach facilitates the exploration of how the combined effects of principal components contribute to predicting whether a sample corresponds to normal lung tissue or lung tumor tissue based on gene expression profiles specific to lung cancer.

## Results
The raw gene expression data exhibits a mean of approximately 23.92 with a wide standard deviation of 119.33 while the scaled data centers around a mean of approximately -0.01 with a more restrained standard deviation of 0.97. Both datasets have similar medians, with the raw data ranging from 0 to 3459.58 and the scaled data ranging from -3.57 to 5.46. The raw data displays a skewness of approximately 12.33, indicating a highly skewed distribution, while the scaled data has a skewness of approximately 0.72. Similarly, the kurtosis of the raw data is approximately 200.84, suggesting heavy-tailedness, whereas the scaled data has a kurtosis of approximately 1.13, indicating a less extreme tail behavior. Overall, scaling gene expression data standardized the values, making them more comparable by removing differences in scale and distribution, while preserving the underlying statistical properties.

![Volcano Plot](/Biomedical%20Probability%20and%20Statistics/Images/volcano.png)

Figure 1. Volcano plot of gene expression. Downregulated genes are showcased in blue, upregulated genes in red, and normally regulated genes in gray. Significance was determined through the application of Welch's t-test and corrected using Bonferroni correction methods. Dotted lines at Log2 Fold Change -1.5 and 1.5 delineate the boundaries for normal regulation.

Among the gene expression data, 7.24% of genes exhibit downregulation, 12.07% display upregulation, while the majority, accounting for 80.68%, show normal regulation. Notably, some of the most significantly regulated genes include BUB1B, HMGA1, STIL, FEN1, and KNL1. The dataset underwent further refinement using insights from the volcano plot, wherein genes with non-significant expression and regulation (p-value > 7.1e-5 and log2 fold change between -1.5 and 1.5) were removed. Significance was determined through the application of a Welch’s t-test, with the Bonferroni correction applied to account for multiple testing.

![Gene Expression Heatmap](/Biomedical%20Probability%20and%20Statistics/Images/heatmap.png)

Figure 2. Heatmap of gene expression created using the R Pheatmap package. Gene expression is color-coded to reflect the degree of regulation, with downregulated genes appearing in varying shades of blue and upregulated genes in shades of red. Patient IDs are annotated with a prefix, "N" for normal patients and "T" for tumor patients. 

The heatmap reveals a distinct separation between normal (denoted by prefix “N”) and tumor patients (denoted by prefix “T”), visually indicating a predominance of upregulated gene expression in tumor samples for the most significantly regulated genes. Clustering of patients was automated by enabling the Pheatmap cluster_col parameter [8]. 

![PCA Plot](/Biomedical%20Probability%20and%20Statistics/Images/pca.png)

Figure 3. 2D PCA-plot showing clustering of “Normal” and “Tumor” diagnoses. Normal diagnoses are represented with blue circles and tumor diagnoses are represented with red triangles.

After collinear variables were identified and removed, PCA was performed on the scaled dataset. The PCA analysis reveals that the first principal component (PC1) explains a significant 37.6% of the dataset's variance, indicating it captures a major pattern distinguishing between different biological states. Although the second principal component (PC2) accounts for a smaller portion of the variance (6.4%), it still plays a crucial role in differentiating data points, likely reflecting variations not captured by PC1. The leading variables impacting PC1, which are BUB1B, HMGA1, KNSTRN, EZH2, and FEN1, correspond to the same genes previously identified as significant, demonstrating the effectiveness of this Principal Component Analysis.

Using the PCA components, several binary logistic regression models were created to forecast gene expression outcomes. The initial model incorporated PCA components PC1 and PC2 as independent variables, whereas the second model exclusively utilized PC1. To compare the models, an ANOVA test was employed with the likelihood ratio test (LRT) option. The resulting p-value was greater than 0.05 (pr(>Chi) = 0.982). Consequently, we failed to reject the null hypothesis, which implies that a model employing just one PCA component (PC1) is preferable to a model that includes more PCA components. This suggests that the primary component alone captures sufficient information for predictive modeling.

Using the reduced model, the estimated coefficient for the intercept is 14.404 with a standard error of 5.205. The z-value of 2.77 and a corresponding p-value of .0057 indicate that the intercept is significantly different from zero at the 5% significance level. The estimated coefficient for PC1 is 1.056 with a standard error of 0.346. For PC1, the z-value is 3.055 with a p-value of 0.0023, which suggests that PC1 is a statistically significant predictor of 'Diagnosis. The positive coefficient indicates that higher values of PC1 are associated with higher log odds of the outcome variable being “Tumor”. The substantial decrease from the null deviance to the residual deviance (from 318.793  to 14.482) further suggests that PC1 has a strong effect in predicting 'Diagnosis'. The odds ratio for PC1 is given by exp(coefficient), which is exp(1.056) = 2.875. Indicating that for a one-unit increase in gene expression for PC1, the odds of being diagnosed with LUSC are about 2.88 times higher.

## Discussion
Studies involving identified genes, BUB1B, HMGA1, KNSTRN, EZH2, and FEN1, show they have been associated with LUSC diagnosis. BUB1B, a key component of the spindle assembly checkpoint, has been implicated in promoting cell proliferation and tumor progression in LUSC [9]. HMGA1, a high-mobility group protein, is known to regulate gene expression and has been found to be overexpressed in LUSC, where it contributes to tumor growth and metastasis [10]. KNSTRN, a kinetochore-associated protein, has been implicated in chromosome segregation and has shown aberrant expression patterns in various cancers, including LUSC. Its dysregulation may contribute to genomic instability and tumor progression in LUSC [11]. Overexpression of EZH2 has been observed in LUSC, where it promotes tumor cell proliferation, invasion, and metastasis [12]. FEN1, a DNA repair enzyme, is involved in maintaining genomic stability, and dysregulation of FEN1 expression has been observed in LUSC, correlating with poor patient outcomes [13]. 

In this analysis, well-known genes associated with LUSC such as FLRT3, PPP2R2C, and MMP3 were not identified as highly significant factors, despite their established roles in the disease. This could be due to various reasons, including limitations in sample size and experimental design, stringent statistical thresholds, and the exclusion of collinear variables. Additionally, the complex nature of LUSC biology may involve genetic pathways not captured in our analysis. Subtle gene expression may still have potent effects in LUSC progression, which would not be identified in this study. Therefore, further investigation using complementary approaches is warranted to fully elucidate the molecular mechanisms underlying LUSC.

## Conclusion
In conclusion, this analysis provides compelling evidence that certain gene expressions are critical in distinguishing between normal tissue and lung squamous cell carcinoma tumor tissues. The application of PCA yielded a robust principal component (PC1) that explains a substantial portion of the variance in gene expression profiles, and this component's significance is reinforced by the logistic regression analysis. PC1 alone has demonstrated its predictive power, simplifying the model without sacrificing accuracy, as supported by the ANOVA test results. The findings support the hypothesis that mutations in specific genes are linked to cancer development, with genes such as BUB1B, HMGA1, KNSTRN, EZH2, and FEN1 standing out in their association with LUSC. The odds ratio derived from the logistic regression model quantifies this risk, indicating a nearly threefold increase in the likelihood of LUSC diagnosis with a unit increase in PC1 gene expression. Further research is needed to explore the functional implications of the identified genes and their potential as therapeutic targets in LUSC.

## References

1. **National Cancer Institute.** (2018). *Cancer of Any Site - Cancer Stat Facts.* SEER. [https://seer.cancer.gov/statfacts/html/all.html](https://seer.cancer.gov/statfacts/html/all.html)

2. **Youlden, D. R., Cramb, S. M., & Baade, P. D.** (2008). *The International Epidemiology of Lung Cancer: Geographical Distribution and Secular Trends.* Journal of Thoracic Oncology, 3(8), 819–831. [https://doi.org/10.1097/jto.0b013e31818020eb](https://doi.org/10.1097/jto.0b013e31818020eb)

3. **Barbar, J., Armach, M., Hodroj, M. H., Assi, S., El Nakib, C., Chamseddine, N., & Assi, H. I.** (2022). *Emerging genetic biomarkers in lung adenocarcinoma.* SAGE open medicine, 10, 20503121221132352. [https://doi.org/10.1177/20503121221132352](https://doi.org/10.1177/20503121221132352)

4. **Ma, X., Ren, H., Peng, R., Li, Y., & Ming, L.** (2020). *Identification of key genes associated with progression and prognosis for lung squamous cell carcinoma.* PeerJ, 8, e9086. [https://doi.org/10.7717/peerj.9086](https://doi.org/10.7717/peerj.9086)

5. **Yang P.** (2009). *Epidemiology of lung cancer prognosis: quantity and quality of life.* Methods in molecular biology (Clifton, N.J.), 471, 469–486. [https://doi.org/10.1007/978-1-59745-416-2_24](https://doi.org/10.1007/978-1-59745-416-2_24)

6. **ÖZÇELIk, B.** (2024, March 4). *TCGA - LUSC: Lung cancer gene expression dataset.* Kaggle. [https://www.kaggle.com/datasets/noepinefrin/tcga-lusc-lung-cell-squamous-carcinoma-gene-exp?resource=download](https://www.kaggle.com/datasets/noepinefrin/tcga-lusc-lung-cell-squamous-carcinoma-gene-exp?resource=download)

7. **Genomics Commons Data Portal.** *Portal.gdc.cancer.gov.* Retrieved February 5, 2024, from [https://portal.gdc.cancer.gov/projects/TCGA-LUSC](https://portal.gdc.cancer.gov/projects/TCGA-LUSC)

8. **Creating heatmaps in R.** (n.d.). [https://igordot.github.io/tutorials/heatmaps-2017-07.nb.html](https://igordot.github.io/tutorials/heatmaps-2017-07.nb.html)

9. **Cicirò, Y., Ragusa, D., & Sala, A.** (2024). *Expression of the checkpoint kinase BUB1 is a predictor of response to cancer therapies.* Scientific Reports, 14(1), 4461. [https://doi.org/10.1038/s41598-024-55080-y](https://doi.org/10.1038/s41598-024-55080-y)

10. **Chen, J., Ji, K., Gu, L., Fang, Y., Pan, M., & Tian, S.** (2022). *HMGA1 Promotes Macrophage Recruitment via Activation of NF-κB-CCL2 Signaling in Hepatocellular Carcinoma.* Journal of Immunology Research, 2022, 1–10. [https://doi.org/10.1155/2022/4727198](https://doi.org/10.1155/2022/4727198)

11. **Zhang, W., Liao, Y., Liu, C., Liu, L., & Zhou, X.** (2023). *KNSTRN, a Poor Prognostic Biomarker, Affects the Tumor Immune Microenvironment and Immunotherapy Outcomes in Pan-Cancer.* Disease Markers, 2023, 1–32. [https://doi.org/10.1155/2023/6729717](https://doi.org/10.1155/2023/6729717)

12. **Fan, K., Zhang, B.-H., Han, D., & Sun, Y.-C.** (2023). *EZH2 as a prognostic-related biomarker in lung adenocarcinoma correlating with cell cycle and immune infiltrates.* BMC Bioinformatics, 24(1). [https://doi.org/10.1186/s12859-023-05271-7](https://doi.org/10.1186/s12859-023-05271-7)

13. **Zhang, Y., Liu, X., Liu, L., Chen, J., Hu, Q., Shen, S., Zhou, Y., Chen, S., Xue, C., Cui, G., & Yu, Z.** (2020). *Upregulation of FEN1 Is Associated with the Tumor Progression and Prognosis of Hepatocellular Carcinoma.* Disease Markers, 2020, 1–17. [https://doi.org/10.1155/2020/2514090](https://doi.org/10.1155/2020/2514090)

